templates/network-manager/fallback-dhcp.nmconnection

```diff
@@ -2,11 +2,16 @@
 id=DHCP on unbound interfaces
 uuid=d1ae9f0b-c8a7-42aa-a238-8f1e9668f5ef
 type=ethernet
+interface-name=eth0
+autoconnect=yes
 autoconnect-priority=-98
-permissions=
+autoconnect-retries=2
 
 [ipv4]
 method=auto
+may-fail=no
+dhcp-timeout=2
```

Comment:
```suggestion
dhcp-timeout=2
```
Could be to short, if users have a DHCP-relay Server, that will fail... Let's think about 15-20?

Reply:
Would be fine with me. Nevertheless this must be documented in our release notes / changes, as this is the delay after the LL fallback will settle.

Reply:
PR is updated